README.md

```diff
@@ -10,36 +10,41 @@ the command line (CLI). There are already excellent GUI based flashing tools.
 This toolbox is for people wanting a complete self-contained lightweight CLI
 solution enabling large scale flashing of headless workers.
 
-This toolbox was written to help manage the [gohci](
-https://github.com/periph/gohci) workers, so they can be reflashed easily in
+This toolbox was written to help manage the [gohci](https://github.com/periph/gohci) workers, so they can be reflashed easily in
 case of tampering. That said, the tools are intentionally generic and reusable.
 
-
 # efe
 
 `efe` is what you run on your workstation to flash one SDCard:
 
-- Fetches the latest OS image for the specified board.
-- Makes a working copy, then modifies the EXT4 root partition to run
+* Fetches the latest OS image for the specified board.
+* Makes a working copy, then modifies the EXT4 root partition to run
   [setup.sh](#setupsh) upon the first boot.
-- Flashes this modified image to the SDCard.
-- Mounts the SDCard, then modifies the FAT32 boot partition to include
+* Flashes this modified image to the SDCard.
+* Mounts the SDCard, then modifies the FAT32 boot partition to include
   `setup.sh` and other data, like `authorized_keys` and settings like the Wifi
   credentials, country, time zone, etc.
 
 It does so **without** requiring any third party software or requiring any UI
 application. It is completely self-contained.
 
+## Installation
+
+Prerequisite: you need to have [go](https://golang.org/dl/) installed on your local machine.
+
+Install this tool:
 
-## Example
+`go install periph.io/x/bootstrap/cmd/...`
 
-This example downloads the [latest Raspbian Stretch Lite image](
-https://www.raspberrypi.org/downloads/raspbian/) and flashes it to an SDCard
+## Usage
+
+This example downloads the [latest Raspbian Stretch Lite image](https://www.raspberrypi.org/downloads/raspbian/) and flashes it to an SDCard
 connected to the workstation. It setups the wifi and sends an email to you when
 it is done.
 
+Note that once the device is booted up, the setup takes a few minutes, the hostname will be changed to `raspberrypi-XXXX` and the email sent to you might be landing in your spam folder.
```

Comment:
you can s/few/several/ to set expectations. :)

wrap at 80 cols

the hostname depends on the board, so it's not necessarily raspberrypi.